packages/react-storage/src/components/StorageBrowser/actions/handlers/types.ts

```diff
@@ -37,3 +38,15 @@ export interface LocationData {
    */
   type: LocationType;
 }
+
+export interface LocationData {
+  bucket: string;
+  permission: Permission;
+  prefix: string;
+  type: LocationType;
+}
+
+export interface TaskHandlerCallbacks
+  extends Pick<TaskHandlerOptions, 'onComplete' | 'onError'> {
+  onCancel?: (key: string) => void;
+}
```

Comment:
We don't need this

Reply:
we use it in the utils, do you mean we should remove it all together? Kept this from your implementation since i was not sure how it affects the rest

Reply:
Oh i see, you just moved it. It's getting deleted in https://github.com/aws-amplify/amplify-ui/pull/6003